content/articles/how-to-create-a-user-login-page-using-javagui/index.md

```diff
@@ -0,0 +1,136 @@
+### How to create a user login page using Java GUI
+
+### Introduction
+
+Java is an Object-Oriented Programming Language. G.U.I stands for Graphical User Interface. GUIs are made up of objects and classes that can interact with each other. GUIs are said to be `Event Driven` programs. this means that GUIs depends on series of object events that can occur at any given time.
+
+### Goals
+
+In this tutorial, we will learn how to design a simple user login page. At the end of this tutorial, the reader will be able to design the user login page below.
+
+![Login page](/how-to-create-a-user-login-page-using-java-gui/loginPage.png)
+
+### Prerequisites
+
+1. Basic Understanding of Java and Java GUIs.
+2. [JDK version 16 or higher](www.oracle.org).
+3. Java compiler of your choice ( Recommended: [Eclipse](www.eclipse.org) )
+
+### Setup procedure
+
+Open eclipse, click on File, select New, then select Class, name your class by Java naming convention.
+If done properly you will have a window like this with the given class name, in this tutorial we name our class Java_GUI.
+
+![java gui](/how-to-create-a-user-login-page-using-java-gui/structure.png)
+**NOTE:** We will import `javax.swing.*;`,`import java.awt.event.*;` this will cover the rest of our java swing extension classes and event package.
+
+#### Variables
+Now let's create our private static variables class and also set it to implements an action whenever the button is click.
+
+```java
+public class Java_GUI implements ActionListener {
+
+	private static JLabel password1, label;
+	private static JTextField username;
+	private static JButton button;
+	private static JCheckBox checkbox;
+	private static JPasswordField Password;
+```
+#### JPanel
+Next, we create a JPanel container which will help in laying out components on the window. But before we can use JPanel we have to import its package.
+
+```java
+//creating a JPanel class
+JPanel panel = new JPanel();
+panel.setLayout(null);
+```
+
+#### JFrame
+
+Now let's import the JFrame extension package and create a JFrame class that will contain all the components that we will see on the window.
+
+```java
+ // JFrame class
+1 JFrame frame = new JFrame();
+2 frame.setTitle("LOGIN PAGE");
+3 frame.setLocation(new Point(500, 300));
+4 frame.add(panel);
+5 frame.setSize(new Dimension(400, 200));
+6 frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
Remove these line numbers.

The readers would find it difficult to copy the codes